uninstall.sh

```diff
@@ -2,12 +2,11 @@
 
 if [ "$(id -u)" -eq 0 ]
 then
-  service log2ram stop
-  systemctl disable log2ram
-  rm /etc/systemd/system/log2ram.service
+  systemctl stop log2ram.service log2ram-daily.timer
+  systemctl disable log2ram.service log2ram-daily.timer
+  rm -rf /etc/systemd/system/log2ram*
```

Comment:
can we do better than a wildcard ? It's only for 2 know file here.

Reply:
Actually, potentially 3.
If you happen to run `systemctl edit log2ram-daily.timer` as I've added in the README, a file named `/etc/systemd/system/log2ram.timer.d/override.conf` would be created containing the user configuration.
The wildcard has been chosen to remove **any** trace of the service from a systemd PoV, including such an overriding.

Tell me what you think about that :ok_hand: